tools/api-markdown-documenter/src/documentation-domain-to-html/ToHtml.ts

```diff
@@ -92,6 +94,10 @@ export function documentationNodeToHtml(
 ): HastTree {
 	const context = getContext(configOrContext);
 
+	if (phrasing(node)) {
```

Comment:
Note for reviewers: now that the documentation domain supports `mdast` content directly (for now, just phrasing content), we special case that content here and just invoke `mdast-util-to-hast`. Eventually this entire transformation layer will be removed in favor of just letting consumers use `mdast-util-to-hast` themselves.